Makefile

```diff
@@ -15,11 +15,10 @@ vendor: glide.yaml
 	# To build without Docker just run "glide install -strip-vendor"
 	docker run --rm \
     -v $(CURDIR):/go/src/github.com/$(PACKAGE_NAME):rw \
-    -v $(HOME)/.glide:/home/user/.glide:rw \
```

Comment:
2 reasons:
 - glide has a bug where  it doesn't update to latest versions of branches if "good enough" is found in cache. Therefore updates should happen with clean cache. Since there is no dedicated target for updates, I figures correctness is better than speed.
 - mounting home dirs doesn't work with Bash on Windows :)